launch/launch/launch_service.py

```diff
@@ -89,8 +89,9 @@ def __init__(
         self.__debug = debug
 
         # Setup logging
-        self.__logger = logging.getLogger('launch.LaunchService')
-
+        self.__logger = logging.getLogger('launch')
```

Comment:
Why did you change the logger name here? Just to be shorter? I'm not necessarily for one or the other, I'm just curious because it seems aesthetic.

Reply:
Purely aesthetic :)

Reply:
That's fine, just curious.